src/views/JsonPView.php

```diff
@@ -2,9 +2,13 @@
 
 class JsonPView extends JsonView
 {
+    /** @var string */
+    protected $callback;
+
     public function __construct($callback)
     {
-        $this->_callback = $callback;
+        parent::__construct();
```

Comment:
That's not necessary at all, as there's no constructor defined in one of the parents.

Additionaly it's not related to the TypeHints, so I'd rather move it to a different PR

Reply:
See: #432